source/js/buyers-guide/search.js

```diff
@@ -15,10 +15,63 @@ const categoryTitle = document.querySelector(`.category-title`);
 const parentTitle = document.querySelector(`.parent-title`);
 const toggle = document.querySelector(`#product-filter-pni-toggle`);
 const subcategories = document.querySelectorAll(`.subcategories`);
+const subContainer = document.querySelector(`.subcategory-header`);
 
 // TODO: turn this into a static class rather than plain JS object.
 const SearchFilter = {
   init: () => {
+    let pos = { left: 0, x: 0 };
+
+    const markScrollStart = (event) => {
+      event.preventDefault();
+      event.stopImmediatePropagation();
+      subContainer.classList.add("cursor-grabbing");
```

Comment:
let's cache this classList at the start of init, such as `const subClasses = subContainer.classList`, so we don't need `subContainer.classList` everywhere. Also, `classList.add()` is an infinite-argument function, so you only need a single `.add()` with as many classes as you need added.